processing/src/main/java/org/apache/druid/query/filter/Filter.java

```diff
@@ -75,13 +77,13 @@ default ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
    * with reasonable sacrifice of the accuracy.
    * As a result, the estimated selectivity might be different from the exact value.
    *
-   * @param indexSelector Object used to retrieve bitmap indexes
+   * @param indexSelector Object used to retrieve indexes
    *
    * @return an estimated selectivity ranging from 0 (filter selects no rows) to 1 (filter selects all rows).
    *
-   * @see Filter#getBitmapIndex(BitmapIndexSelector)
+   * @see Filter#getBitmapIndex(ColumnIndexSelector)
    */
-  double estimateSelectivity(BitmapIndexSelector indexSelector);
+  double estimateSelectivity(ColumnIndexSelector indexSelector);
```

Comment:
I didn't explicitly mention it, but I was opting to refactor selectivity estimation as a follow-up change, since this was already pretty big, so I'd prefer a follow-up I think